charters/wot-ig-2019.html

```diff
@@ -215,8 +215,8 @@ <h2>Scope</h2>
             <p style="text-align:center;font-weight:bold;">Scope Summary</p>
             <ul>
                 <li>Support the Web of Things Working Group</li>
-                <li>Organize and run PlugFests to evaluate the current working assumptions in regards to Web of Things</li>
-                <li>Reach out and collaborate with interested organizations, vendors, and communities in support of the Interest Group's mission</li>
+                <li>Organize and run PlugFests, including PlugFests for specific vertical industries, to evaluate the current working assumptions in regards to the Web of Things</li>
```

Comment:
suggest to reword:
> including PlugFests for specific vertical industries

to

> including interoperability events for specific vertical industries

Reply:
How about "Organize and run interoperability and testing events, including events for specific vertical industries, to evaluate..."

Reply:
Group agrees to my suggested change (main call).